assets/src/edit-story/components/tablist/tablistPanel.js

```diff
@@ -81,11 +76,15 @@ const TablistPanel = ({
           <PanelText aria-hidden>{badgeCount}</PanelText>
         </Badge>
       </TabButton>
-      <Scrollable maxHeight={maxHeight}>
-        <TabPanel aria-labelledby={`${title}-${panelId}`} role="tabpanel">
+      <ScrollableContent maxHeight={maxHeight}>
+        <TabPanel
+          id={panelId}
```

Comment:
Needed this to make sure the button was identifying the content it's controlling with `aria-controls`